### DISCUSSION TITLE
@ckeditor/ckeditor5-utils: Options.target's type is incomplete

### DISCUSSION THREAD

**Discussion by neongreen** [DefinitelyTyped/DefinitelyTyped#54517] [Issues with a @types package] [answered]
The current type in dom/position.d.ts is:

```
    /**
     * Target with respect to which the `element` is to be positioned.
     */
    target: HTMLElement | Range | ClientRect | Rect | (() => HTMLElement);
```

However, CKEditor just always calls `target` as a function if it's a function, so I think the right type is:

```
    target: HTMLElement | Range | ClientRect | Rect | (() => HTMLElement | Range | ClientRect | Rect);
```

cc @fedemp

**Comment by fedemp** [comment, marked as answer]
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/54581

**Reply by fedemp** [reply]
@neongreen Could you check if the issue was resolved by https://github.com/DefinitelyTyped/DefinitelyTyped/pull/54581 and mark as answered if possible?

**Reply by neongreen** [reply]
I can't test it now but it looks good

**Reply by fedemp** [reply]
Thank you. :)
